app/locustio/common_utils.py

```diff
@@ -109,31 +146,12 @@ def global_measure(func, start_time, *args, **kwargs):
         result = func(*args, **kwargs)
     except Exception as e:
         total = int((time.time() - start_time) * 1000)
-
-        # Delete this workaround after fix from Taurus.
-        for handler in events.request_failure._handlers:
-            argspec = inspect.getfullargspec(handler)
-            if argspec.varkw is None and 'self' in argspec.args:
-                # Special case for incompatible Taurus handler
-                handler(request_type='Action',
-                        name=f"locust_{func.__name__}",
-                        response_time=total,
-                        exception=e)
-            else:
-                handler(
-                    request_type='Action',
-                    name=f"locust_{func.__name__}",
-                    response_time=total,
-                    exception=e,
-                    response_length=0,
-                )
-
-        # Uncomment with fix of __on_failure() function from Taurus. Expected Taurus version with the fix is 1.14.3
-        # events.request_failure.fire(request_type="Action",
-        #                             name=f"locust_{func.__name__}",
-        #                             response_time=total,
-        #                             response_length=0,
-        #                             exception=e)
+        print(e)
```

Comment:
Does this print go to bzt.log or to locust.log?

Reply:
This print goes to locust.log

Reply:
maybe it worth to add extra tesing to make clear what this `e` is about.